CHANGES.rst

```diff
@@ -3149,7 +3147,7 @@ Changelog (Pillow)
 - Change function declaration to match Tcl_CmdProc type #1966
   [homm]
 
-- Integer overflow checks on all calls to *alloc #1781
```

Comment:
Note that this change changes the meaning of the line.

Reply:
Thanks for the review, Do you mean we need to show * before alloc? That was an error in RST. @wiredfool 
So `*alloc`?

Reply:
Yes, `*alloc`, thanks